test/karma.conf.js

```diff
@@ -0,0 +1,50 @@
+var path = require('path');
+
+module.exports = function(config) {
+  config.set({
+    basePath: '../',
+    browsers: ['PhantomJS'],
+    coverageReporter: {
+      type: 'lcov',
+      dir: 'test/coverage/'
+    },
+    files: [
+      'test/bind-polyfill.js',
+      'packages/*/test/**/*.js',
+    ],
+    frameworks: ['mocha', 'chai', 'sinon-chai'],
+    reporters: ['mocha', 'coverage'],
+    preprocessors: {
+      'packages/**/*.js': ['webpack']
+    },
+    webpack: {
+      resolve: {
+        extensions: ['', '.js', '.jsx'],
+        alias: {
+          helpers: path.join(__dirname, 'helpers'),
+          fixtures: path.join(__dirname, 'fixtures'),
+          'component-playground': path.join(__dirname, '../packages/react-component-playground/src')
+        }
+      },
+      module: {
+        loaders: [{
+          test: /\.jsx?$/,
+          exclude: /node_modules/,
+          loader: 'babel',
```

Comment:
I made a loader that wraps each describe into a describe with file path, you can use it: https://www.npmjs.com/package/describe-loader

```
// npm install --save-dev describe-loader

postLoaders: [{
    test: /\.js$/,
    exclude: /node_modules/,
    loader: 'describe'
  }]
```